src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -9478,6 +9478,7 @@ std::pair<std::string,std::string> SketchObject::getElementName(
     index.appendToStringBuffer(ret.second);
     if (auto realName = convertInternalName(ret.second.c_str())) {
         Data::MappedElement mappedElement;
+        (void)realName;

```

Comment:
Why did you decide not to use `boost::ignore_unused` here as well? (It's new to me, I've never used it before)

Reply:
Because below there is commented some code which I guess will be re-activated in the future.